dashboard-for-web/articles/web-dashboard-designer-mode/appearance-customization/conditional-formatting.md

```diff
@@ -27,9 +23,7 @@ Format rules used in conditional formatting can be categorized as follows:
 * **Bar Color Ranges** - Visualize numeric values as bars. Values within a range display a specific color.
 * **Bar Gradient Ranges** - Visualize numeric values as bars. Values within a range display a specific color.
```

Comment:
The description here seems to be a duplicate of Line 23. Should this description be:

`Visualize numeric values as bars. Values within a range display a specific bar gradient.`